bin/grade_item.py

```diff
@@ -68,6 +68,25 @@ def add_permissions_recursive(top_dir,root_perms,dir_perms,file_perms):
             add_permissions(os.path.join(root, f),file_perms)
 
 
+
+def get_vcs_info(top_dir,semester,course,gradeable,userid):
+
+    form_json_file = os.path.join(top_dir,"courses",semester,course,"config","form","form_"+gradeable+".json")
+
+    with open(form_json_file, 'r') as fj:
+        form_json = json.load(fj)
+
+    is_vcs=form_json["upload_type"]=="repository"
+    vcs_type="git"
+    vcs_base_url=""
+    vcs_subdirectory=""
+    if is_vcs:
+        vcs_subdirectory= form_json["subdirectory"]
+
+    vcs_subdirectory = vcs_subdirectory.replace("{$user_id}",userid)
```

Comment:
I'd like to have a consistent use of spacing on setting variables (preferably `a = b`)?